mackup/applications/amethyst.cfg

```diff
@@ -3,3 +3,4 @@ name = Amethyst
 
 [configuration_files]
 Library/Preferences/com.amethyst.Amethyst.plist
+.config/amethyst/amethyst.yml
```

Comment:
It needs to go under an `xdg_configuration_files` section, I'll fix it.

Reply:
https://github.com/lra/mackup/pull/1906/commits/146d5b25d6be981b40c599b9ac0ff0453c884d8b